tools/example-templates/additional-files/Attr3Attribute.cs

```diff
@@ -0,0 +1,5 @@
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
```

Comment:
This one *does* have a file extension, but is still the same as Attr1...

Reply:
See previous reply